Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -118,9 +147,62 @@ final class RewardsCollectionViewController: UICollectionViewController {
       .observeValues { [weak self] project, reward, refTag in
         self?.goToDeprecatedPledge(project: project, reward: reward, refTag: refTag)
       }
+
+    self.viewModel.outputs.rewardsCollectionViewFooterIsHidden
+      .observeForUI()
+      .observeValues { [weak self] isHidden in
+        self?.updateRewardCollectionViewFooterConstraints(isHidden)
+      }
+
+    self.viewModel.outputs.configureRewardsCollectionViewFooterWithCount
+      .observeForUI()
+      .observeValues { [weak self] count in
+        self?.updateRewardsCollectionFooterView(with: count)
+      }
+
+    self.viewModel.outputs.flashScrollIndicators
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.collectionView.flashScrollIndicators()
+      }
+
+    self.rewardsCollectionFooterView.rac.hidden = self.viewModel.outputs.rewardsCollectionViewFooterIsHidden
   }
 
-  // MARK: - Private Helpers
+  // MARK: - Functions
+
+  private func setupConstraints() {
+    _ = self.collectionView
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+
+    NSLayoutConstraint.activate([
+      self.rewardsCollectionFooterView.leftAnchor.constraint(equalTo: self.view.leftAnchor),
```

Comment:
Yeah I think usually you want to use `leading` and `trailing` for positioning labels. `left` and `right` seems accurate here because we're positioning the footer relative to the absolute `left` and `right` positions of the superview.

Reply:
Ok yeah makes sense to just do it for labels.